app/src/components/account/UserMenu.vue

```diff
@@ -123,8 +147,13 @@
 <script>
 import Avatar from "common/Avatar"
 import UserMenuAddress from "account/UserMenuAddress"
+import { AddressRole } from "src/gql"
```

Comment:
feels like it should go in session.js or keystore.js

Reply:
yes, I am going to change this here

Reply:
Done